run_all.py

```diff
@@ -0,0 +1,39 @@
+# run with `python3 ../run_all.py`  (must be in build dir)
+
+import os
+import sys
+
+# checks if cwd is build
+if os.getcwd().split("/")[2] != "build":
```

Comment:
This is definitely platform- and possibly machine-specific. Use `os.path.split(os.getcwd())[1]` instead.